project/Rat.scala

```diff
@@ -39,22 +39,6 @@ object Rat {
     file("daffodil-runtime2/src/test/c/examples"),
 
     // test files that cannot include the Apache license without breaking tests
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/982"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1326"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1328"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1329"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1330"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1331"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1333"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1334"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1337"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1338"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1339"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1340"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1382"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1591"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1602"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/debugger/1863"),
```

Comment:
Does the PR delete these files because their contents had already been inlined as input strings into the corresponding CLI tests in an earlier PR?  Nothing seems to read or use these files even before this PR was created, so it's time to remove them.

Reply:
Looking at the commit history, when debugger tests were originally added (nearly 10 years) ago they used the `-d` option and passed in these files. A short time later (04e50f9aad93eae15a02d7eff4b9440e84bc9e2d), we switched to ExpectIt and changed these tests to inline the debugger commands, and I guess we just forgot to remove the files.